README.md

```diff
@@ -1,9 +1,28 @@
 ocaml-unix-unistd
-================
+=================
 
 [ocaml-unix-unistd](https://github.com/dsheets/ocaml-unix-unistd) provides
-host-dependent unistd.h access.
+access to the features exposed in [`unistd.h`][unistd.h] in a way that is not
+tied to the implementation on the host system.
 
-**WARNING**: not portable due to *read*, *close*, *access*, *symlink*,
-*readlink*, *truncate*, and *ftruncate* wrappers that assume 64-bit
-instruction pointers.
+The [`Unistd`][unistd] module provides functions for translating between the
+constants accessible through `unistd.h` and their values on particular
+systems.
+
+The [`Unistd_unix`][unistd_unix] module provides bindings to functions that
+use the constants and types in `Unistd` along with a representation of the
+host system.  The bindings support a more comprehensive range of seek commands
+than the corresponding functions in the standard OCaml `Unix` module.  The
+[`Unistd_unix_lwt`][unistd_unix_lwt] module exports non-blocking versions of
+the functions in `Unistd_unix` based on the [Lwt][lwt] cooperative threading
+library.
+
+**WARNING**: the current implementation is not portable due to `read`,
+`write`, `pread`, and `pwrite` wrappers that assume 64-bit instruction
+pointers.
```

Comment:
It's true for the moment.  [Here's an example](https://github.com/dsheets/ocaml-unix-unistd/blob/4666dba2f43436b55967c37f0aaab4d628bc2db5/unix/unistd_unix.ml#L151).